doc/versionHistory.md

```diff
@@ -0,0 +1,5 @@
+# Version History
+
+0.0.1
+
+- Add README.md, versionHistory.md, moduleRelays.md, moduleDaq, and moduleIlc.md.
```

Comment:
Add `README.md`, `versionHistory.md`, `moduleRelays.md`, `moduleDaq.md`, and `moduleIlc.md`.
The final two are the placeholders at this moment.

Reply:
Done.